### PR TITLE
Added RN versions on README file

### PR DESCRIPTION
**Pull request by gabceb**
## Description of changes

Added section on README with RN versions as reassurance that the module works as expected for new devs and those updating to new RN version

## Related issues (if any)

N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +63,12 @@ A good starting point for troubleshooting is:
 
 If you're still having trouble, post an issue so we can look into it.
 
+## Tested with the following RN versions
+
+Folks using this module have confirmed the library works with the following React Native versions. If your version is not on the list and you have confirmed it works as expected please create a PR (you can even do it on github.com)
+
+- 0.42.0
```
Comment (AlanFoster) [inline README.md]: Confirmed working with 0.43.x and 0.44.x :+1:

I understand that as of React 0.46.x there is a breaking change to the bundler, but I'm not sure if react-native-schemes-manager will have any issues with this upgrade yet.

Full context: https://github.com/facebook/react-native/releases/tag/v0.46.4

Comment (blargity) [inline README.md]: If they just moved the file around, then it won't matter, as we bundle our own version. I'll keep an eye on this an ensure it continues to work post 0.46.

**Comment by blargity** [comment]
Thank you!

File: README.md (hunk 1 above)
Comment (AlanFoster) [inline README.md]: Thanks for the sanity check; Confirmed working with RN 0.48 - and now live in production :+1:

**Pull request merged**